customers/schema.py

```diff
@@ -107,6 +107,11 @@ class BaseProfileFieldsMixin:
     berth_applications = DjangoConnectionField(BerthApplicationNode)
     berth_leases = DjangoConnectionField(BerthLeaseNode)
 
+    def resolve_berth_applications(self, info, **kwargs):
+        qs = BerthApplication.objects
+
+        return qs.filter(customer=self.id).order_by("created_at")
```

Comment:
you don't haave to initiate a totally new queryset with this `filter(customer=self.id)`, you can just do `return self.berth_applications.order_by("created_at")`. reverse FK relationships are available at this level, and in fact without this custom resolver that it exactly what graphene does: `return self.<field_name>.all()`  (or in this case `return self.berth_applications.all()`)

Reply:
👍 will try it out

Reply:
works like a charm